tests/bdd/higher_order_gherkin.py

```diff
@@ -45,7 +45,7 @@ def create_step_file(cls, source_dir, target_file):
 
     @classmethod
     def _scan_dir(cls, target_dir):
-        for feature_file in glob(target_dir + "*.feature"):
+        for feature_file in glob(target_dir + "/*.feature"):
```

Comment:
I think `importlib_resources` might strip trailing slashes, which caused this to smoosh together